binder/environment.yml

```diff
@@ -6,8 +6,8 @@ channels:
 dependencies:
 - pip
 - idr-py
-- scikit-image=0.17.*
-- scikit-learn=0.23.*
+- scikit-image
```

Comment:
I think we should pin to a newer version otherwise this could lead to some unexpected failure